dashboard/test/models/user_level_test.rb

```diff
@@ -291,14 +296,14 @@ class UserLevelTest < ActiveSupport::TestCase
 
     script_level = create :script_level, levels: [@level], lesson: stage
 
-    ul_student = UserLevel.create(user: @user, level: @level, submitted: true)
-    ul_teacher = UserLevel.create(user: teacher, level: @level, submitted: true)
+    ul_student = UserLevel.create(user: @user, level: @level, locked: true)
+    ul_teacher = UserLevel.create(user: teacher, level: @level, locked: true)
 
     assert_equal true, script_level.locked?(@user)
     assert_equal false, script_level.locked?(teacher)
 
-    assert_equal true, ul_student.locked?(stage)
-    assert_equal false, ul_teacher.locked?(stage)
+    assert_equal true, ul_student.show_as_locked?(stage)
```

Comment:
look how much clearer these two assertions are now that the method is named `show_as_locked?`! :)